src/solvers/dgmulti/sbp.jl

```diff
@@ -302,7 +302,9 @@ Constructs a single-element [`VertexMappedMesh`](@ref) for a single periodic ele
 a DGMulti with `approximation_type` set to a periodic (finite difference) SBP operator from
 SummationByPartsOperators.jl.
 """
-function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS}) where {NDIMS}
+function CartesianMesh(dg::DGMultiPeriodicFDSBP{NDIMS};
+                       coordinates_min=ntuple(_ -> -1.0, NDIMS),
+                       coordinates_max=ntuple(_ -> 1.0, NDIMS)) where {NDIMS}
```

Comment:
I guess this will become another name in #1012 as discussed there?

Reply:
Yes - let me change that in this PR.